### PR TITLE
Google Code In experience 2017-2018

### PR DESCRIPTION
**Pull request by w0lF07**
This is a blog about my experience in GOOGLE CODE IN

Google code-in is a world-wide competition held by the folks at Google annually which provides a platform for us to explore new and interesting organizations and also help them! With tasks in all branches of cyber-computing such as: Code, User Interface, Documentation/Training, Quality Assurance and Outreach/Research it is the ultimate competition. It mainly aims at open source.

I am Swastik Das, a  student of class XI at SAI International School, Bhubaneswar, India.
I heard about Google Code In from Sampriti Panda who has participated in GCI trice earlier and has also been the Grand prize Winner last year.
From childhood computer have fascinated me. I love everything related to them let it be gaming, editing stuffs or coding.
This year I decided to participate in the GCI.
I chose JBoss Community on reading that it is related to Java.
I got to learn about many open-source softwares. They are great to work with. You can use them and also modify and upgrade those softwares which is a great thing.
I learnt how to develop apps, integrate authentication methods, designing webpages and various other stuffs.
At present I know Java, Javascript, C++, C#, Swift, HTML, CSS and I would love to learn more and master in these programming languages too.

GCI gave me a nice opportunity to improve my coding skills as well as learn many new things. It also gave me new friends and supporting mentors who ere always there to guide us. Whether I get prize or not, I ll forever remember such valuable experience because learning is more important than winning.

### CODE REVIEW

File: _posts/2018-01-01-gciblog.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+I learnt how to develop apps, integrate authentication methods, designing webpages and various other stuffs.
+At present I know Java, Javascript, C++, C#, Swift, HTML, CSS and I would love to learn more and master in these programming languages too.
+
+##Mentors
+The mentors are really good and cooperative.
+They check all our works and also advice us on improving our programming skills and also to rectify to mistakes.
+They are really friendly and never step back in clearing doubts or helping in any other ways.
+
+
+
```
Comment (yashovardhanagrawal) [inline _posts/2018-01-01-gciblog.md]: unnecessary spaces here, please delete them

**Comment by yashovardhanagrawal** [comment]
@garg-anuj build is failing here, please see to it

**Comment by w0lF07** [comment]
@yashovardhanagrawal done now. Any more changes??

**Comment by yashovardhanagrawal** [comment]
can you squash your commits?

**Pull request closed without merging**